### ISSUE TITLE
Crash with 1.12.2 forge 2718+ 

### ISSUE THREAD

**Issue by ClockwerkKaiser** [Chisel-Team/ConnectedTexturesMod#63]
I'm getting this crash whenever I try to access the inventory of any Immersive Engineering storage crates. 

```
---- Minecraft Crash Report ----
// I blame Drullkus, author of CTM.

Time: 2018-06-28 09:33:58 EDT
Description: Rendering item

java.lang.ClassCastException: it.unimi.dsi.fastutil.ints.Int2ObjectOpenHashMap cannot be cast to gnu.trove.map.hash.TIntObjectHashMap
    at team.chisel.ctm.client.model.ModelUtil.lambda$getMesh$0(ModelUtil.java:51)
    at java.util.Optional.map(Unknown Source)
    at team.chisel.ctm.client.model.ModelUtil.getMesh(ModelUtil.java:51)
    at team.chisel.ctm.client.model.AbstractCTMBakedModel$Overrides.handleItemState(AbstractCTMBakedModel.java:84)
    at sun.reflect.GeneratedMethodAccessor50.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at Reflector.call(Reflector.java:955)
    at net.minecraft.client.renderer.RenderItem.getItemModelWithOverrides(RenderItem.java:398)
    at codechicken.lib.render.item.CCRenderItem.getItemModelWithOverrides(CCRenderItem.java:258)
    at codechicken.lib.render.item.CCRenderItem.renderItemAndEffectIntoGUI(CCRenderItem.java:210)
    at net.minecraft.client.gui.inventory.GuiContainer.drawSlot(GuiContainer.java:265)
    at net.minecraft.client.gui.inventory.GuiContainer.drawScreen(GuiContainer.java:98)
    at blusunrize.immersiveengineering.client.gui.GuiIEContainerBase.drawScreen(GuiIEContainerBase.java:28)
    at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:368)
    at sun.reflect.GeneratedMethodAccessor28.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at Reflector.callVoid(Reflector.java:648)
    at net.minecraft.client.renderer.EntityRenderer.updateCameraAndRender(EntityRenderer.java:1456)
    at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1117)
    at net.minecraft.client.Minecraft.run(Minecraft.java:3938)
    at net.minecraft.client.main.Main.main(SourceFile:123)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
    at java.lang.reflect.Method.invoke(Unknown Source)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

I updated forge earlier, and tested a few versions. It's seems to happen with forge version 14.23.4.2718 and up. I'm assuming it has something to do with the changes below. 

```
Build 1.12.2-14.23.4.2718:
	tterrag1098: Fix MC-234: Z-fighting when viewing break animation side-on (#4928)
	latvianmodder: Added Item#readNBTShareTag(ItemStack, NBTTagCompound) (#4932)
	bs2609: Switch from trove to fastutil collections (#4933)
	UnRealDinnerbone: Add Farmland Trample Event  (#4895)
```

Not sure if this is on IE or CTM, so I'll post this to both.

**Comment by ChloeDawn** [comment]
CTM reflects into Forge classes which were changed in a recent build, breaking CTM's reflection. Please downgrade to `1.12.2-14.23.4.2717` until this is resolved.

**Comment by ClockwerkKaiser** [comment]
Will do! Thank you!

**Comment by CluelessPhil** [comment]
This bug happened to me when I was in creative and I clicked on Thermal Series: Blocks

**Comment by tterrag1098** [comment]
Fixed in 0.3.1
